mappings/net/minecraft/world/LightType.mapping

```diff
@@ -1 +1,4 @@
 CLASS bhr net/minecraft/world/LightType
+	FIELD c LIGHT I
```

Comment:
What's this constant? Maybe missing context but `LIGHT` seems pretty vague

Reply:
Luminance better?

Reply:
How is it used?

Reply:
It seems to be just used in LightType class, in the constructor

Reply:
It's not a `static final` constant, it should be camelCase. The value doesn't seem to be used anywhere (unless Enigma is reporting wrong) I think it's best to not map it.